code/modules/jobs/job_types/roguetown/nobility/captain.dm

```diff
@@ -137,26 +136,34 @@
 	H.set_blindness(0)
 	switch(weapon_choice)
 		if("Zweihander")
+			H.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
 			r_hand = /obj/item/rogueweapon/greatsword/zwei
 			backl = /obj/item/gwstrap
 		if("Great Mace")
+			H.adjust_skillrank(/datum/skill/combat/maces, 1, TRUE)
 			r_hand = /obj/item/rogueweapon/mace/goden/steel
 		if("Battle Axe")
+			H.adjust_skillrank(/datum/skill/combat/axes, 1, TRUE)
 			r_hand = /obj/item/rogueweapon/stoneaxe/battle
 		if("Greataxe")
+			H.adjust_skillrank(/datum/skill/combat/axes, 1, TRUE)
 			r_hand = /obj/item/rogueweapon/greataxe/steel/doublehead
 			backl = /obj/item/gwstrap
 		if("Estoc")
+			H.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
 			r_hand = /obj/item/rogueweapon/estoc
 			backl = /obj/item/gwstrap
 		if("Longsword & Shield")
+			H.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
 			r_hand = /obj/item/rogueweapon/sword/long
 			beltr = /obj/item/rogueweapon/scabbard/sword
 			backl = /obj/item/rogueweapon/shield/tower/metal
 		if("Flail & Shield")
+			H.adjust_skillrank(/datum/skill/combat/whipsflails, 1, TRUE)
 			beltr = /obj/item/rogueweapon/flail/sflail
 			backl = /obj/item/rogueweapon/shield/tower/metal
 		if("Sabre & Shield")
+			H.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
```

Comment:
use
`H.adjust_skillrank_up_to(/datum/skill/, 5, TRUE)`
for all of these please, this will secure it from virtue exploits